app/services/idv/steps/document_capture_step.rb

```diff
@@ -58,6 +62,15 @@ def acuant_sdk_upgrade_a_b_testing_variables
         }
       end
 
+      def in_person_cta_variant_testing_variables
+        bucket = AbTests::IN_PERSON_CTA.bucket(flow_session[:document_capture_session_uuid])
```

Comment:
Where is the following AC met?
> Logging set up to correctly measure and analyze the two variations

Reply:
Specifically I would like to know if you identified a place in the code where the variant is logged, and also if you identified whether such logging can be easily correlated with the existing analytics events (identifying steps of the in-person proofing process) that are currently being sent to CloudWatch.

Reply:
Ok, just reviewed the Google Doc and saw the variant being logged in the `success_state` field. If that's the case then the main question remaining is whether the current approach to logging the variant sufficiently facilitates correlation for the A/B test.

Reply:
I think I need more training on our logging system to be able to more thoughtfully answer your question. What I do know:
- When a user session is "bucketed", that variant value gets added to the InPersonContext
- When a variant view is rendered for the user, the variant is logged with `trackEvent`. A sample log line:
`{
  "name": "Frontend: IdV: IPP CTA Variant B",
  "properties": {
    "event_properties": {
      "flow_path": "standard",
      "acuant_sdk_upgrade_a_b_testing_enabled": "false",
      "use_alternate_sdk": "false",
      "acuant_version": "11.7.1"
    },
    "new_event": true,
    "new_session_path": false,
    "new_session_success_state": true,
    "success_state": "POST|/api/logger|Frontend: IdV: IPP CTA Variant B",
    "path": "/api/logger",
    "session_duration": 2881.533648,
    "user_id": "c149fa41-03c9-48dd-a5d6-3dd5a98ffea2",
    "locale": "en",
    "user_ip": "::1",
    "hostname": "localhost",
    "pid": 1602,
    "service_provider": null,
    "trace_id": null,
    "git_sha": "ff939530",
    "git_branch": "jess/LG-8274-IPP-CTA-AB-test-higher-level-switching",
    "user_agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/109.0.0.0 Safari/537.36",
    "browser_name": "Chrome",
    "browser_version": "109.0.0.0",
    "browser_platform_name": "macOS",
    "browser_platform_version": "10.15.7",
    "browser_device_name": "Unknown",
    "browser_mobile": false,
    "browser_bot": false
  },
  "time": "2023-01-23T20:56:06.533Z",
  "id": "a60e7160-a463-4d9e-bf60-9240311ffc28",
  "visitor_id": "cc0db1dc-17ef-4341-b46d-975e8c63bac6",
  "visit_id": "c28108eb-f163-4056-8998-93a5c7fc86e8"
}
`

Reply:
This PR as-is would permit us to identify users who saw a specific variant and subsequently look at a few individual users' activity. This would not permit us to automatically and easily query, graph, or monitor the aggregate activity of those users after they saw the variant.

We could get part of the way there by running one query to get the users, then making a subsequent query with the IDs. That could be costly or untenable for large numbers of users, though, and requires ongoing manual effort.

Some alternatives would be:
- Include the variant seen by the user in subsequent doc auth/IPP session logs
  - **This is the solution I prefer and recommend.**
- Rely on rough correlation instead of direct knowledge
  - This reduces data quality, so would require Product buy-in.
- Prevent variant overlap by setting different ones to 100% at different times
  - This runs counter to the current AC.

Reply:
> - Include the variant seen by the user in subsequent doc auth/IPP session logs

This would set the variant directly in the user's session, and logs later in the process would be adjusted as needed to ensure that the variant gets logged. You can find many of the existing types of logs in [`analytics.rb`](https://github.com/18F/identity-idp/blob/main/app/services/analytics.rb).

Reply:
The variant is static for a given session ID; can we not coalesce on that value to correlate the variant with subsequent events of interest?

Reply:
I don't see documentation indicating that CloudWatch supports [the statistic we're using](https://github.com/18F/identity-idp/blob/e8cd6b3149b4a6f90fc2105a5b416b171a099c94/lib/ab_test_bucket.rb#L31) for either [CW Insights](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/CWL_QuerySyntax.html) nor [CW Metrics](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/using-metric-math.html).

Reply:
Very good information! Thank you!

Reply:
Sumi and I landed on releasing this logging as-is with a high priority ticket to follow which addresses these concerns. Thanks @NavaTim for spending so much time on clarifying the issues here !!